src/__tests__/test.js

```diff
@@ -41,20 +39,24 @@ describe('ml-expectation-maximization test', function () {
         };
     }
 
-    it('Example with 500 points', function () {
+    test('Example with 500 points', function () {
         var result = getModel();
-        var em = result.model, groups = result.groups;
+        var em = result.model;
+        var groups = result.groups;
 
         var clusterData = em.getClusterData();
 
         clusterData.sort(function (a, b) {
             return a.weight - b.weight;
         });
 
-        for(var i = 0; i < groups.length; ++i) {
-            clusterData[i].weight.should.be.approximately(groups[i].weight, 0.1);
-            Matrix.sub(clusterData[i].mean, groups[i].mu).abs().sum().should.be.approximately(0, 0.3);
-            clusterData[i].should.have.properties(['mean', 'covariance', 'prediction', 'weight']);
+        for (var i = 0; i < groups.length; ++i) {
+            expect(clusterData[i].weight).toBeCloseTo(groups[i].weight, 1);
+            expect(Matrix.sub(clusterData[i].mean, groups[i].mu).abs().sum()).toBeCloseTo(0, 0);
```

Comment:
here you are giving it a range of ±1, is that the idea?

Reply:
yes, because I'm generating points with a given mean and a random generator between [0, 1]